fastlane/env/user.env-example

```diff
@@ -2,6 +2,12 @@ GHHELPER_ACCESS=
 APPCENTER_API_TOKEN=
 SENTRY_AUTH_TOKEN=
 
-# Token to trigger builds via the CircleCI APIs
-# See: https://circleci.com/docs/2.0/managing-api-tokens/
-CIRCLE_CI_AUTH_TOKEN=<CircleCI Auth Token>
+# Fastlane Match
+MATCH_S3_ACCESS_KEY=<S3 Access Key>
+MATCH_S3_SECRET_ACCESS_KEY=<S3 Secret Access Key>
+MATCH_PASSWORD=<Fastlane Match Password>
+
+# App Store Connect API
+APP_STORE_CONNECT_API_PZYM8XX95Q_KEY_ID=<ASC Key ID>
+APP_STORE_CONNECT_API_PZYM8XX95Q_ISSUER_ID=<ASC Key Issuer ID
+APP_STORE_CONNECT_API_PZYM8XX95Q_KEY_CONTENT=<ASC Key Content>
```

Comment:
If you go with my suggestion above to use the env vars expected by the `app_store_connect_api_key` action's `ConfigItem` instead of custom ones, this will allow to run `app_store_connect_api_key` without any parameter and thus make the call site in the `Fastfile` way less verbose 🙂

Reply:
Updated in https://github.com/Automattic/simplenote-macos/commit/87b404ef0c480d9a4c846dc85882d5cfc55aaf21